spec/rails_helper.rb

```diff
@@ -69,6 +70,15 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+
+  #before every test, webmock will stub our API call to the movie/top_rated endpoint
+  config.before do 
+    WebMock.stub_request(:any, "https://api.themoviedb.org/3/movie/top_rated?api_key=#{ENV["moviedb_key"]}")
+           .to_return(body: File.read('./spec/fixtures/top_movies.json'))
+
+    WebMock.stub_request(:any, "https://api.themoviedb.org/3/search/movie?api_key=#{ENV["moviedb_key"]}&query=river")
+           .to_return(body: File.read('./spec/fixtures/search_river_movies.json'))
+  end
```

Comment:
This is cool, and looks easy enough for me to implement as well. Is the :any argument necessary, I want my stub to fail if I am accidentally making a call that is not permitted. When I was writing my stubs, it would error if I tried to create any api calls that I didn't stub / permit.